include/geode/geosciences/representation/io/cross_section_input.h

```diff
@@ -45,26 +45,18 @@ namespace geode
     CrossSection opengeode_geosciences_geosciences_api load_cross_section(
         absl::string_view filename );
 
-    class opengeode_geosciences_geosciences_api CrossSectionInput : public Input
+    class CrossSectionInput : public Input< CrossSection >
```

Comment:
is it still possible to derivate from `CrossSectionInput`? I mean, I do not understand why removing the decl.

Reply:
Everything is now implemented in the header file. So I think the export is no longer needed...